src/mmw/js/src/core/settings.js

```diff
@@ -3,7 +3,8 @@
 var defaultSettings = {
     itsi_embed: false,
     base_layers: {},
```

Comment:
Oh I see we're using it as an associative array, so an empty object is the correct initialization. Never mind.